docs/releases/5.2.txt

```diff
@@ -389,6 +389,17 @@ Dropped support for PostgreSQL 13
 Upstream support for PostgreSQL 13 ends in November 2025. Django 5.2 supports
 PostgreSQL 14 and higher.
 
+Models
+------
+
+* Multiple changes have been made to the undocumented ``django.db.models.sql.compiler.SQLCompiler.execute_sql``
```

Comment:
Alright, I'll just remove it. I do feel like it's probably hard to end up in a triangulation of "custom compiler" + "test coverage misses something that would cause a problem here" + "people who are not used to digging through the internals".